pagopa/pn-frontend
# Refactor/PN-11356 - Remove unused code

## Short description
This PR clean the codebase removing no more used code and models

## List of changes proposed in this pull request
- removes Statistics.api.ts deprecated after bff integration
- removes all code related to the rejected DigitalAttemptsStatistics component and changes statistics models accordingly
- removes unused redux/statistics/types.ts

## How to test
Prerequisite: IS_STATISTICS_ENABLED env var should be declared and set to true in config.json
- Navigate to the statistics section and verify all the functionalities keep working as expeceted.
